sys/behave/behave.cabal

```diff
@@ -67,10 +67,11 @@ test-suite behave-test
   build-depends:       base
                      , containers
                      , mtl
+                     , text
+
                      , HUnit
                      , MissingH
                      , text
```

Comment:
Already on line 70

Reply:
Two different items: test-suite (line 44) and library (line 15)

Reply:
No, same file line 70 and 74. Check the last line up there, under MissingH.

Reply:
removed in deadcode pull request!